tutorials/Bangle.js Locale.md

```diff
@@ -16,108 +16,122 @@ into Bangle.js's firmware, but you can install the `Language` app from [the App
 correctly in the console. They may need to be rendered with
 [a 6x8 or other ISO10646-1 font](/Fonts).
 
 ```JS
 // The name of the Locale
->require('locale').name
+>require("locale").name
 ="de_DE"
 
 // Symbol used for currency
->require('locale').currencySym
+>require("locale").currencySym
 ="€"
 
 // Day of the week from Date (long)
->require('locale').dow(new Date())
+>require("locale").dow(new Date()) // or dow(new Date(), 0)
 ="Donnerstag"
 
-// Day of the week from Date  (short)
->require('locale').dow(new Date(), 1)
+// Day of the week from Date (short)
+>require("locale").dow(new Date(), 1)
 ="Do"
 
+// Day of the week from Date (first letter only, uppercase)
+>require("locale").dow(new Date(), 2)
+="D"
+
+// All the days of the week (long)
+>require("locale").dowAll() // or dowAll(0)
+=[ "Sonntag", "Montag", "Dienstag", "Mittwoch", "Donnerstag", "Freitag", "Samstag" ]
+
+// All the days of the week (short)
+>require("locale").dowAll(1)
+=[ "So", "Mo", "Di", "Mi", "Do", "Fr", "Sa" ]
+
+// All the days of the week (first letter only, uppercase)
+>require("locale").dowAll(2)
+=[ "S", "M", "D", "M", "D", "F", "S" ]
+
 // Month of the year from Date (long)
->require('locale').month(new Date())
+>require("locale").month(new Date())
 ="April"
 
 // Month of the year from Date (short)
->require('locale').month(new Date(), 1)
+>require("locale").month(new Date(), 1)
 ="Apr"
 
 // Number to formatted String
->require('locale').number(4.98)
+>require("locale").number(4.98)
 ="4.98"
 
 // Currency to formatted String
->require('locale').currency(4.98)
+>require("locale").currency(4.98)
 ="4.98€"
 
 // Distance/Length (in meters) to formatted String
->require('locale').distance(150)
+>require("locale").distance(150)
 ="150m"
 
 // Automatic unit selection for larger distances
->require('locale').distance(15000)
+>require("locale").distance(15000)
 ="15km"
 
 // Speed (in kph) formatted String
->require('locale').speed(150)
+>require("locale").speed(150)
 ="150kmh"
 
 // Temperature (in Centigrade) to String
->require('locale').temp(150)
+>require("locale").temp(150)
 ="150°C"
 
 // Translate test or a phrase
->require('locale').translate("off")
+>require("locale").translate("off")
 ="aus"
 
 // Date to date string (long)
->require('locale').date(new Date())
+>require("locale").date(new Date())
 ="Donnerstag, 02. April 2020"
 
 // Date to date string (short)
->require('locale').date(new Date(),1)
+>require("locale").date(new Date(), 1)
 ="02.04.2020"
 
 // Date to time string (long)
->require('locale').time(new Date())
+>require("locale").time(new Date())
 ="15:49:39"
 
 // Date to time string (long)
->require('locale').time(new Date(),1)
+>require("locale").time(new Date(), 1)
 ="15:49"
 
 // Date to meridian (text describing morning/evening)
->require('locale').medidian(new Date())
+>require("locale").medidian(new Date())
 ="" // or "pm" for en_GB
 ```
 
-Translation
-------------
+Translations (deprecated)
```

Comment:
Might be worth mentioning  `/*LANG*/` here as the non-deprecated option

Reply:
Ok! I can add some info about the `LANG` stuff :)

Reply:
I'll add some documentation about the new `date_utils` also.